README.md

```diff
@@ -1,16 +1,18 @@
-# LaunchDarkly Sample Xamarin Application s
+# LaunchDarkly Sample Client-Side .NET Applications
 
-We've built a simple demo that demonstrates how LaunchDarkly's SDK works. Since the Xamarin SDK can be used either on Xamarin-compatible mobile devices or in portable .NET code, there are three versions of the demo: an Android app, an iOS app, and a .NET Core console app.
+We've built a simple demo that demonstrates how LaunchDarkly's SDK works. Since the client-side .NET SDK can be used either on Xamarin-compatible mobile devices or in portable .NET code, there are three versions of the demo: a Xamarin Android app, a Xamarin iOS app, and a .NET Core console app.
 
-Below, you'll find the basic build procedures, but for more comprehensive instructions, you can visit your [Quickstart page](https://app.launchdarkly.com/quickstart#/) or the [Xamarin SDK reference guide](https://docs.launchdarkly.com/sdk/client-side/xamarin).
+Important: these demos are for the _client-side_ .NET SDK, which is suitable for mobile or desktop applications. For server-side use, see https://github.com/launchdarkly/hello-dotnet-server.
```

Comment:
I've already renamed the `hello-dotnet` repo to `hello-dotnet-server`. There may still be some links elsewhere to the old name but they will redirect automatically.